src/language/lexer.ts

```diff
@@ -692,7 +702,11 @@ function readName(
   );
 }
 
-// _ A-Z a-z
+/**
+ * ```
+ * _ A-Z a-z
+ * ```
+ */
```

Comment:
It's not exported function please leave it as is

Reply:
the issue here is that eslint complains. So might need to look in how to escape it for non-exported members